cli/src/commands/federated-graph/commands/fetch.ts

```diff
@@ -21,6 +22,10 @@ export default (opts: BaseCommandOptions) => {
       },
     );
 
+    if (resp.response?.code === EnumStatusCode.ERR_FREE_TRIAL_EXPIRED) {
+      program.error(resp.response.details || 'Free trial has concluded. Please talk to sales to upgrade your plan.');
```

Comment:
Can we put them into a funnel directly? Link to sales?

Reply:
DIscussed on added the cal link